Src/AutoFixture.xUnit.net.UnitTest/DelegatingFixture.cs

```diff
@@ -101,5 +102,7 @@ public object Create(object request, ISpecimenContext context)
         }
 
         internal Func<object, ISpecimenContext, object> OnCreate { get; set; }
+
+        internal Action<ICustomization> OnCustomize { get; set; }

```

Comment:
If I understand the idea with `Func` correctly, the test could look like the following:
```csharp
var fixture = new DelegatingFixture();
fixture.OnCustomize = c =>
{
    c.Customize(fixture);
    return fixture;
};
//...
Assert.False(fixture.Customizations[0] is FreezeOnMatchCustomization);
Assert.True(fixture.Customizations[1] is FreezeOnMatchCustomization);
```

I tried to build it this way originally but then I realized that the `fixture.Customizations` property is of type `IList<ISpecimenBuilder>` rather than `IList<ICustomization>`. And the order of the *Specimen Builder*'s might not correspond to the original *Customizations* order. That is why I decided to introduce the `customizationLog` list.